gemviz/bluesky_runs_catalog_table_model.py

```diff
@@ -13,12 +13,21 @@
 import pyRestTable
 import yaml
 from PyQt5 import QtCore
+from PyQt5 import QtGui
 
 from . import analyze_run
 from . import utils
 
 DEFAULT_PAGE_SIZE = 20
 DEFAULT_PAGE_OFFSET = 0
+BGCLUT ={  # BackGround Color Lookup Table
+    "success": None,
+    # mark background color of unsuccessful runs
+    "abort": QtGui.QColorConstants.Svg.lightyellow,  # ffffe0
+    "fail": QtGui.QColorConstants.Svg.mistyrose,  # ffe4e1
+    # other, such as None (when no stop document)
+    "other": QtGui.QColorConstants.Svg.lightslategrey,  # 778899
```

Comment:
I find this color very dark, it makes it stands out way to much. I would make it much lighter, like maybe `#dce0e5`? (much lighter but still very different from the standard Qt color). Or another color entirely (light blue?)
Also, what is the meaning of `other` in that context? If this is just because there is no exist status, which could mean anything (could be successful for all we know?), do we really want to highlight them?

Reply:
Color choice is fine.  We should mark failed runs with No exit status or runs with unexpected exit status.  That's how the `other` is used.

Reply:
If learn of a new status that should be regarded as successful, we can add that to the `BGCLUT` dictionary. https://github.com/BCDA-APS/gemviz/blob/020883d9f3dd380a8c015833b69a67b490be9c27/gemviz/bluesky_runs_catalog_table_model.py#L23-L30